packages/client/bin/cli.ts

```diff
@@ -199,8 +206,68 @@ async function run() {
   // give network id precedence over network name
   const chain = args.networkId ?? args.network ?? 'mainnet'
 
-  // configure common based on args given
+  // configure accounts for mining and prefunding in a local devnet
+  const accounts: [Address, Buffer][] = [] // [address, privateKey]
+  if (args.unlock) {
+    const rl = readline.createInterface({
+      input: process.stdin,
+      output: process.stdout,
+    })
+    const question = (text: string) => {
+      return new Promise<string>((resolve) => {
+        rl.question(text, resolve)
+      })
+    }
+
+    try {
+      for (const addressString of args.unlock) {
+        const address = Address.fromString(addressString)
+        const inputKey = await question(
+          `Please enter the 0x-prefixed private key to unlock ${address}:\n`
+        )
+        const privKey = toBuffer(inputKey)
+        const derivedAddress = new Address(privateToAddress(privKey))
+        if (address.equals(derivedAddress)) {
+          accounts.push([address, privKey])
+        } else {
+          throw new Error(
+            `Private key does not match for ${address} (address derived: ${derivedAddress})`
+          )
+        }
+      }
+    } catch (error) {
+      console.error(`Encountered error unlocking account:\n${error}`)
+    }
+    rl.close()
+  }
+
   let common: Common = {} as Common
+  if (args.dev) {
+    fs.rmdirSync('./dev', { recursive: true })
```

Comment:
That could make sense.  If we want to allow that option (as opposed to a truly ephemeral devnet), I'll set it up so it only deletes the folder if there's no mining account/private key provided.  Otherwise, the client won't start because the genesis state will be different when a new mining account gets generated.